lib/KeyboardAwareMixin.js

```diff
@@ -112,7 +114,8 @@ const KeyboardAwareMixin = {
   /**
    * @param extraHeight: takes an extra height in consideration.
    */
-  scrollToFocusedInput: function (reactNode: Object, extraHeight: number = this.props.extraHeight) {
+  scrollToFocusedInput: function (reactNode: Object, extraHeight: number) {
+    extraHeight = extraHeight || this.props.extraHeight;
```

Comment:
This can yield to unexpected results if `extraHeight` is `0`, I prefer to check if the parameter is actually passed, not rely on its falsy value.

Can you change this? I'm happy to merge the rest of the code :)